LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -12,53 +12,82 @@ import LoopKit
 import SwiftUI
 
 public struct TherapySettingsView: View, HorizontalSizeClassOverride {
+    public struct ActionButton {
+        public init(localizedString: String, action: @escaping () -> Void) {
+            self.localizedString = localizedString
+            self.action = action
+        }
+        let localizedString: String
+        let action: () -> Void
+    }
+    
     @Environment(\.dismiss) var dismiss
         
     @ObservedObject var viewModel: TherapySettingsViewModel
     
     @State var isEditing: Bool = false
     
-    private let mode: PresentationMode
-    
-    public init(mode: PresentationMode = .settings, viewModel: TherapySettingsViewModel) {
-        self.mode = mode
+    private let actionButton: ActionButton?
+        
+    public init(viewModel: TherapySettingsViewModel,
+                actionButton: ActionButton? = nil) {
         self.viewModel = viewModel
+        self.actionButton = actionButton
     }
-    
+        
     public var body: some View {
-        switch mode {
-        case .acceptanceFlow, .settings: return AnyView(content)
-        case .legacySettings: return AnyView(navigationContent)
+        switch viewModel.mode {
+        case .acceptanceFlow: return AnyView(content)
+        case .settings: return AnyView(contentWithNavigationButtons)
+        case .legacySettings: return AnyView(navigationViewWrappedContent)
         }
     }
     
     private var content: some View {
         List {
-            if viewModel.prescription != nil {
-                prescriptionSection
-            }
-            suspendThresholdSection
-            correctionRangeSection
-            temporaryCorrectionRangesSection
-            basalRatesSection
-            deliveryLimitsSection
-            insulinModelSection
-            carbRatioSection
-            insulinSensitivitiesSection
-            if viewModel.includeSupportSection {
-                supportSection
+            Group {
```

Comment:
[design question] When I built the overview screen, I noticed there aren't lines between the individual settings - is that intentional? (the design currently includes them)

Reply:
Build
<img width="362" alt="Screen Shot 2020-07-24 at 4 40 09 PM" src="https://user-images.githubusercontent.com/31571514/88443271-81078e80-cdcc-11ea-96b4-f1bbd3002969.png">

Design
<img width="598" alt="Screen Shot 2020-07-24 at 4 40 26 PM" src="https://user-images.githubusercontent.com/31571514/88443301-9a103f80-cdcc-11ea-9640-05a29fb51208.png">

Reply:
@novalegra I'm pretty sure I'm doing whatever the default is.  I can ask at standup today, but since this is unrelated to this PR, can I please get an approval?  I can make a followup PR if there's a need...

Reply:
Sure!